File: FertiWeb/src/main/java/dto/DTORecomendacion.java
package dto;

import java.util.Date;
import java.util.List;

import lombok.Data;

@Data
public class DTORecomendacion {

	private Long codigoRecomendacion;
	private DTOAnalisis analisis;
	private Date fechaRecomendacion;
	private List<DTOFuenteXRecomendacion> recomendacionXFuente;

}

File: FertiMobile/src/app/dto/dto-recomendacion.ts
import { DTOFuenteXRecomendacion } from './dto-fuente-x-recomendacion';
import { DTOAnalisis } from './dto-analisis';

export class DTORecomendacion {
    codigoRecomendacion: number;
    analisis: DTOAnalisis;
    fechaRecomendacion: string;
    recomendacionXFuente: DTOFuenteXRecomendacion[];
}
